ui/app/templates/components/auth-info.hbs

```diff
@@ -54,7 +54,7 @@
           {{/if}}
         {{/if}}
         <li class="action">
-          <LinkTo @route="vault.cluster.logout" @model={{activeClusterName}} @id="logout" class="is-destroy" @invokeAction={{onLinkClick}}>
+          <LinkTo @route="vault.cluster.logout" @model={{@activeClusterName}} @id="logout" class="is-destroy" @invokeAction={{@onLinkClick}}>
```

Comment:
I noticed elsewhere that you had the action keyword in front of this `@onLinkClick` Maybe confirm that this link is working?

Reply:
Yes, it works. This action is coming from the parent.